tests/test_package_orchestrator.py

```diff
@@ -0,0 +1,179 @@
+        )
+
+        check_installed(setup_tester_ssh_connection, "mender-orchestrator-support")
+        setup_tester_ssh_connection.run(
+            "test -x /usr/share/mender/inventory/mender-inventory-orchestrator-inventory"
+        )
+        setup_tester_ssh_connection.run(
+            "test -x /usr/share/mender/modules/v3/mender-orchestrator-manifest"
+        )
+        setup_tester_ssh_connection.run(
+            "test -x /usr/share/mender-orchestrator/interfaces/v1/rootfs-image"
+        )
+
+        # mender-orchestrator-support-demo
+        upload_deb_package(
+            setup_tester_ssh_connection,
+            mender_dist_packages_versions["mender-orchestrator-support"],
+            "mender-orchestrator-support-demo",
+            "all",
+        )
+        setup_tester_ssh_connection.run(
+            "sudo dpkg --install "
+            + package_filename(
+                mender_dist_packages_versions["mender-orchestrator-support"],
+                "mender-orchestrator-support-demo",
+                "all",
+            )
+        )
+        check_installed(setup_tester_ssh_connection, "mender-orchestrator-support")
+        setup_tester_ssh_connection.run(
+            "test -f /data/mender-orchestrator/topology.yaml"
+        )
+        setup_tester_ssh_connection.run(
+            "test -d /data/mender-orchestrator/mock-instances/0"
+        )
+        setup_tester_ssh_connection.run(
+            "test -d /data/mender-orchestrator/mock-instances/1"
+        )
+        setup_tester_ssh_connection.run(
+            "test -d /data/mender-orchestrator/mock-instances/2"
+        )
+
+
+@pytest.mark.usefixtures("setup_mender_configured")
+class TestPackageOrchestratorMeta:
+    @pytest.mark.commercial
+    def test_mender_orchestrator_meta_package(
+        self, setup_tester_ssh_connection, mender_dist_packages_versions
+    ):
+        # Upload all the packages (including the demo one)
+        for pkg in [
+            "mender-orchestrator",
+            "mender-orchestrator-core",
+        ]:
+            upload_deb_package(
+                setup_tester_ssh_connection,
+                mender_dist_packages_versions["mender-orchestrator"],
+                pkg,
+            )
+        for pkg in [
+            "mender-orchestrator-support",
+            "mender-orchestrator-support-demo",
+        ]:
+            upload_deb_package(
+                setup_tester_ssh_connection,
+                mender_dist_packages_versions["mender-orchestrator-support"],
+                pkg,
+                "all",
+            )
+
+        setup_tester_ssh_connection.run("mkdir -p /packages; mv *.deb /packages/")
+        prepare_local_apt_repo(setup_tester_ssh_connection, "/packages")
+
+        # Install the meta-package only
+        setup_tester_ssh_connection.run(
+            "sudo apt install --assume-yes mender-orchestrator",
+        )
+
+        check_installed(setup_tester_ssh_connection, "mender-orchestrator-core")
```

Comment:
The `mender-orchestrator` meta package should also be installed, right?

Reply:
Yes. I thought that that was obvious and the important was to check the dependencies. I'll add it anyway.